tests/testthat/test-anova.R

```diff
@@ -0,0 +1,15 @@
+context("anova tidiers")
+
+test_that("tidy.aovlist works"){
+  aovlistfit <- aov(mpg ~ wt + disp + Error(drat), mtcars)
+  td <- suppressWarnings(tidy(aovlistfit))
```

Comment:
What kind of warnings are getting suppressed here?

Reply:
I checked and there actually are none. Warning suppression was just a hold over from copying the format of the `test-lm.R`. I'll take it out